docs/trace/stress/Program.cs

```diff
@@ -0,0 +1,124 @@
+﻿// <copyright file="Program.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+using OpenTelemetry.Internal;
+
+public class Program
+{
+    private static long cntEvents = 0;
+
+    public static int Main()
+    {
+        return EntryPoint();
+    }
+
+    private static int EntryPoint()
+    {
+        var cntWriter = Environment.ProcessorCount; // configure the number of writers

```

Comment:
Set `cntWriter` as `Environment.ProcessorCount - 1` and reserve one core for reader?

Reply:
That would make 0 writer on a single core machine 😆

Reply:
That's a good edge case, perhaps set 1 as minimum.😆